.golangci.yml

```diff
@@ -84,24 +134,48 @@ issues:
   - path: pkg/pipelinerunmetrics/injection.go
     linters:
     - containedctx
+  - path: pkg/pod/pod.go
+    linters:
+    - maintidx
   - path: pkg/pod/creds_init_test.go
     linters:
     - containedctx
+  - path: pkg/reconciler/pipelinerun/pipelinerun.go
+    linters:
+    - maintidx
   - path: pkg/taskrunmetrics/injection.go
     linters:
     - containedctx
   - path: test/controller.go
     linters:
     - containedctx
+  - path: internal/sidecarlogresults/sidecarlogresults\.go
```

Comment:
nit: Why do some of these paths have `\.go` and some just have `.go`?

Reply:
Just because we've been sloppy (and I'm likely a big part of that "we"). Cleaned up for consistency, and added an explanatory comment.